Purchases/SwiftInterfaces/RCOperationDispatcher.h

```diff
@@ -13,9 +13,8 @@ NS_ASSUME_NONNULL_BEGIN
 NS_SWIFT_NAME(OperationDispatcher)
 @interface RCOperationDispatcher : NSObject
 
-- (void)dispatchOnMainThreadIfSet:(void (^ _Nullable)(void))block;
 - (void)dispatchOnMainThread:(void (^)(void))block;
-- (void)dispatchOnSameThreadIfSet:(void (^ _Nullable)(void))block;
```

Comment:
I removed the `ifSet` ones because I realized that they're not particularly useful - if a `completion` block is nil, doing `completion(foo, bar` is going to raise an exception, not no-op. but the block itself won't be nil, because you're calling it like so
```objective-c
[operationDispatcher dispatchOnMainThread: ^ {
    completion(foo);
];
```
which makes the block == `^{ completion(foo) }` != nil, even if `completion` == nil